.github/workflows/openapi-diff.yml

```diff
@@ -1,42 +1,62 @@
 name: openapi-diff
 on:
   pull_request:
-    paths:
-      - 'api-docs/**'
 
 jobs:
-    check-diff:
-      runs-on: ubuntu-18.04
-      steps:
-        - name: Checkout getsentry/sentry
-          uses: actions/checkout@v2
-          with:
-            path: sentry
-
-        - name: Getsentry Token
-          id: getsentry
-          uses: getsentry/action-github-app-token@v1
-          with:
-            app_id: ${{ secrets.SENTRY_INTERNAL_APP_ID }}
-            private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
-
-        - name: Checkout getsentry/sentry-api-schema
-          uses: actions/checkout@v2
-          with:
-            ref: 'main'
-            repository: getsentry/sentry-api-schema
-            path: sentry-api-schema
-            token: ${{ steps.getsentry.outputs.token }}
-
-        - name: Install/setup node
-          uses: volta-cli/action@v1
-
-        - name: Build OpenAPI Derefed JSON
-          run: |
-            cd sentry
-            yarn install --frozen-lockfile
-            yarn run build-derefed-docs api-docs/openapi-derefed.json
-
-        - name: Compare OpenAPI Derefed JSON
-          run: |
-            npx json-diff --color sentry-api-schema/openapi-derefed.json sentry/api-docs/openapi-derefed.json
+  check-diff:
+    name: build api
+    runs-on: ubuntu-20.04
+    timeout-minutes: 90
+    strategy:
+      matrix:
+        python-version: [3.8.12]
+
+    steps:
+      - name: Getsentry Token
+        id: getsentry
+        uses: getsentry/action-github-app-token@v1
+        with:
+          app_id: ${{ secrets.SENTRY_INTERNAL_APP_ID }}
+          private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
+
+      - uses: actions/checkout@v2
+        with:
+          # Avoid codecov error message related to SHA resolution:
+          # https://github.com/codecov/codecov-bash/blob/7100762afbc822b91806a6574658129fe0d23a7d/codecov#L891
+          fetch-depth: '2'
+
+      - name: Check for python file changes
```

Comment:
i'm not quite sure how i'm supposed to use this filter -- i'd like to use the ones listed in `api_docs` within file-filters.yml (same for other action)

Reply:
So you'll need to reference this action via `steps.<step ID>`, then its outputs: `steps.changes.outputs`. The action has [these outputs](https://github.com/dorny/paths-filter#outputs) for each filter name. So you'll want something like `steps.changes.outputs.api_docs == 'true'` to see if api docs has changed

Reply:
got it. do I need to add this check to every step thereafter or just the next one? i'm assuming every one.